ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentBaseline.tsx

```diff
@@ -172,7 +172,10 @@ function DeploymentBaselines({ deployment, deploymentId, onNodeSelect }: Deploym
                             <Tooltip
                                 content={
                                     <div>
-                                        Trigger violations for network policies not in the baseline
+                                        Trigger violations for traffic not in the baseline. This
+                                        offers soak time for finalizing your network policies, where
```

Comment:
@boazmichaely @vjwilson I don't hear `soak time` too often. Is that friendly for persons not too familiar with the English language?